tabularreportgenerator.json

```diff
@@ -0,0 +1,166 @@
+    "type": "information",
+    "product_vendor": "Splunk Community",
+    "logo": "logo_tabularreportgenerator.svg",
+    "logo_dark": "logo_tabularreportgenerator_dark.svg",
+    "product_name": "SOAR",
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk Community",
+    "contributors": [
+        {
+            "name": "Daniel Federschmidt"
+        }
+    ],
+    "license": "Copyright (c) Splunk Community, 2022",
+    "app_version": "1.0.0",
+    "utctime_updated": "2022-01-11T12:32:43.321597Z",
+    "package_name": "phantom_tabularreportgenerator",
+    "main_module": "tabularreportgenerator_connector.py",
+    "min_phantom_version": "5.2.0",
+    "app_wizard_version": "1.0.0",
+    "configuration": {
+        "image_list": {
+            "description": "Name of custom list storing images",
+            "data_type": "string",
+            "order": 0
+        }
+    },
+    "pip3_dependencies": {
+        "wheel": [
+            {
+                "module": "dataclasses",
+                "input_file": "wheels/py3/dataclasses-0.8-py3-none-any.whl"
+            },
+            {
+                "module": "pydantic",
+                "input_file": "wheels/py36/pydantic-1.9.0-cp36-cp36m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl"
+            },
+            {
+                "module": "typing_extensions",
+                "input_file": "wheels/py3/typing_extensions-4.1.1-py3-none-any.whl"
+            }
+        ]
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "generate report email",
+            "identifier": "generate_report_email",
+            "description": "Generate raw email output containing the report",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {
+                "report_json": {
+                    "description": "Report specification in JSON format",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "trg report json"
+                    ],
+                    "order": 0
+                },
+                "from": {
+                    "description": "Report specification in JSON format",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 1
+                },
+                "to": {
+                    "description": "Report specification in JSON format",
```

Comment:
Same as above

Reply:
Not doing any changes as per slack conversation.